src/main/scala/com/itechart/ny_accidents/service/MergeService.scala

```diff
@@ -30,16 +30,43 @@ class MergeService @Inject()(weatherService: WeatherMappingService,
           case Some(data) => data
           case None =>
             println("Not in cache")
-            val data = createMergedData(value)
+            val data = fullMergeData(value)
             cacheService.cacheMergedData(data)
             data
         }
-      case None => createMergedData(value)
+      case None => fullMergeData(value)
     }
+  }
+
+  def withoutWeatherMapper(value: Accident): MergedData = {
+    if (counter % 100 == 0)
+      println("COUNTER: " + counter)
```

Comment:
remove println statements please

Reply:
removed

Reply:
I've removed them in NEAF-84, that is a child of this branch